corfu_scripts/corfu_layouts.clj

```diff
@@ -81,7 +81,8 @@ Options:
                                              (try
                                                (.get (.bootstrapLayout (get-layout-client) new-layout))
                                                (catch Exception e
-                                               (println server":" (.getMessage e))))
+                                                 (println server":" (.getMessage e))
+                                                 (throw e)))
```

Comment:
I see the issue.

When there is a new layout proposed, we should ensure that all nodes are : 

1. Already bootstrapped in the same cluster OR
2. Not Bootstrapped yet

It is probably sufficient to test that all new nodes in the layout (deltas) are not already bootstrapped. I have tweaked the code to do that and it works as expected for both scenarios mentioned.

1. If 9001 is not already bootstrapped, it will install the new layout.
2. If 9001 is already bootstrapped, it will just refuse it.

Is there other scenario not taken care of by this behavior?